codebuild/bin/install_default_dependencies.sh

```diff
@@ -45,12 +47,6 @@ fi
 if [[ "$S2N_LIBCRYPTO" == "openssl-1.0.2-fips" ]] && [[ ! -d "$OPENSSL_1_0_2_FIPS_INSTALL_DIR" ]]; then
     codebuild/bin/install_openssl_1_0_2_fips.sh "$(mktemp -d)" "$OPENSSL_1_0_2_FIPS_INSTALL_DIR" "$OS_NAME" ; fi
 
-# Download and Install CppCheck
-if [[ "$BUILD_S2N" == "true" ]]; then
-    mkdir -p "$CPPCHECK_INSTALL_DIR"||true
-    codebuild/bin/install_cppcheck.sh "$CPPCHECK_INSTALL_DIR" > /dev/null ;
-fi
-
 # Download and Install LibreSSL
 if [[ "$S2N_LIBCRYPTO" == "libressl" ]]; then
     mkdir -p "$LIBRESSL_INSTALL_DIR"||true
```

Comment:
Should libre and boringssl follow the same install convention as openssl and gnutls? I saw openssl1.0.2 didn't change either. Are there differences or are the tricky bits handled in a different install file?

Reply:
For consistency sake, agreed.  LibreSSL and Boring installs don't get run multiple times (yet) with the current buildspecs, but it's conceivable we'd add more flags/iterations to them in the future.